repository-slug/README.md

```diff
@@ -0,0 +1,81 @@
+# TLX Repository Gate Problem Slug Style Guide
+
+## Daftar Isi
+
+- [Latar Belakang](#latar-belakang)
+- [Batasan](#batasan)
+- [Soal Kontes](#soal-kontes)
+- [Soal Training](#soal-training)
+- [Soal Lainnya](#soal-lainnya)
+- [Lain-lain](#lain-lain)
+
+## Latar Belakang
+
+[TLX Repository Gate](https://repository.ia-toki.org/) adalah tempat untuk meletakan soal-soal yang digunakan di TLX, baik [TLX Training Gate](https://training.ia-toki.org/) maupun [TLX Competition Gate](https://competition.ia-toki.org/). Sampai dengan 29 Juni 2017 terdapat lebih dari 200 kontes berbeda yang terdapat di TLX Competition Gate yang mencakup Pelatnas TOKI, Open OSN, TOKI Open, TOKI Open Contest, dan lain lain (termasuk kontes yang tidak diselenggarakan oleh TOKI). Kontes-kontes ini memiliki panitia yang berbeda-beda, dan setiap panitia kontes harus meletakan soal-soalnya di TLX Repository Gate.
```

Comment:
Saya agak anti mencantumkan tahun/tanggal/jumlah spesifik di sebuah dokumen style guide, karena seolah-olah maintainer harus terus melakukan update pada statistik ini seiring berjalannya waktu.

Meralat pernyataan saya sebelumnya, bagaimana kalau semua occurrences of 200, 1.000 etc diganti dengan `banyak sekali` ? Jumlah soal tidak terlalu penting; karena yang ditekankan adalah "banyak sekali soal sedemikian sehingga penamaan slug menjadi sangat penting".